be/src/vec/data_types/serde/data_type_ipv6_serde.cpp

```diff
@@ -87,7 +87,10 @@ void DataTypeIPv6SerDe::write_one_cell_to_jsonb(const IColumn& column,
     result.writeKey(col_id);
     IPv6 data = assert_cast<const ColumnIPv6&>(column).get_element(row_num);
     IPv6Value ipv6_value(data);
-    result.writeString(ipv6_value.to_string());
+    std::string ipv6_str = ipv6_value.to_string();
```

Comment:
it may cost too much
Could ipv6 value return StringRef directly?

Reply:
why not store int128 directly?

Reply:
now we can use int128 directly